Rakefile

```diff
@@ -119,3 +119,40 @@ def xcodebuild(command)
     sh "xcodebuild #{command}"
   end
 end
+
+def formatTool(command)
+  # As of Xcode 13.4 / Xcode 14 beta 4, including airbnb/swift as a dependency
+  # causes Xcode to spin indefinitely at 100% CPU (due to the remote binary dependencies
+  # used by that package). As a workaround, we can specifically add that dependency
+  # to our Package.swift file when linting / formatting and remove it afterwards.
+  packageDefinition = File.read('Package.swift')
```

Comment:
Does this work if you execute the rake task from a subdirectory?

Reply:
it does!

Reply:
I'm actually surprised lol, I didn't test it originally and I expected it to not work when I tried it just now